src/main/java/io/glutenproject/integration/tpc/Tpc.java

```diff
@@ -71,6 +72,12 @@ public class Tpc implements Callable<Integer> {
   @CommandLine.Option(names = {"--file-format"}, description = "Option: parquet, dwrf", defaultValue = "parquet")
   private String fileFormat;
 
+  @CommandLine.Option(names = {"--conf"}, description = "Test line Spark conf, separate by ';', if more than one config, should wrap with \"\"", defaultValue = "")
```

Comment:
I am not sure if picocli supports parsing arg value to Java `Map`. Would you like to help investigate on it?

Reply:
Good idea. I have changed to Map and test 3 cases

- without --conf
- --conf=a=b
- --conf=spark.sql.optimizer.runtime.bloomFilter.enabled=true --conf="a=b c"

It works.